Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -580,37 +616,83 @@ class BrowserViewController: UIViewController {
             make.bottom.equalTo(footer.snp.top)
         }
 
-        bottomContentStackView.snp.remakeConstraints { make in
-            make.left.right.equalTo(view)
-            make.centerX.equalTo(view)
-            make.width.equalTo(view.safeArea.width)
+        bottomContentStackView.snp.remakeConstraints  { make in
+            adjustBottomContentStackView(make)
+        }
 
-            // Height is set by content - this removes run time error
-            make.height.greaterThanOrEqualTo(0)
+        adjustBottomSearchBarForKeyboard()
+    }
 
-            if let keyboardHeight = keyboardState?.intersectionHeightForView(view), keyboardHeight > 0 {
-                make.bottom.equalTo(view).offset(-keyboardHeight)
-            } else if !toolbar.isHidden {
-                make.bottom.lessThanOrEqualTo(toolbar.snp.top)
-                make.bottom.lessThanOrEqualTo(view.safeArea.bottom)
-            } else {
-                make.bottom.equalTo(view.safeArea.bottom)
-            }
+    private func adjustBottomContentStackView(_ make: ConstraintMaker) {
+        make.left.right.equalTo(view)
+        make.centerX.equalTo(view)
+        make.width.equalTo(view.safeArea.width)
+
+        // Height is set by content - this removes run time error
+        make.height.greaterThanOrEqualTo(0)
+        bottomContentStackView.setContentHuggingPriority(.defaultHigh, for: .vertical)
+
+        if isBottomSearchBar {
+            adjustBottomContentBottomSearchBar(make)
+        } else {
+            adjustBottomContentTopSearchBar(make)
         }
     }
```

Comment:
Did you mean to remake constraints here instead of `make`?

@lmarceau considering here https://github.com/mozilla-mobile/firefox-ios/pull/9961/files#diff-f100cf547899b755789a923f8dd2af6c047de636f01f1da9b8d95442ef559d3dL547 we are remaking all constraints

Reply:
`make` is coming from remake in that case. I'll rename to make it more explicit:

```
bottomContentStackView.snp.remakeConstraints  { make in
       adjustBottomContentStackView(make)
} 
```